frontend/src/app/hello/routes.ts

```diff
@@ -0,0 +1,15 @@
+import { NgModule } from "@angular/core";
+import { Routes, RouterModule } from "@angular/router";
+import { HelloComponent } from "./hello.component";
+import { DetailComponent } from "./detail/detail.component";
+
+export const routes: Routes = [
+  { path: "", component: HelloComponent },
+  { path: "detail", component: DetailComponent }
+];
+
+@NgModule({
+  imports: [RouterModule.forRoot(routes)],
+  exports: [RouterModule]
+})
+export class RouteModule {}
```

Comment:
I would rename this to `HelloRoutingModule` to stay consistent with naming convention, and the file to _hello-routing.module.ts_.